### ISSUE TITLE
Hashes and Signature for verification??

### ISSUE THREAD

**Issue by 2SI3NX** [bitconnectcoin/bitconnectcoin#6]
Any idea why and where are the hashes and signature/public keys for the file downloads?

Your github page is nowhere to be found on https://bitconnectcoin.co/ as far as ive seen. I had to manually search for this.
Your signatures/hashes/public keys for files are nowhere to be found either.

Surely this is a must requirement when dealing with money just like all other cryptocurrencies have, unless im missing something??

**Comment by FaysalM** [comment]
I completely agree - there are no sigs, and almost every single open source practice is being ignored. The code looks old and does not seem to compile correctly either. Either open up everything like others have or stop calling this open source?

**Comment by 2SI3NX** [comment]
News sites have reported that Bitconnect is not so much of a trustworthy cryptocurrency and there is a suspicion that once they collect lots of peoples money in bitcoins that they might disappear.
Clearly no response from the developers means that they just made a github page just to close peoples eyes and trick them.

**Comment by TechniumUnlimited** [comment]
:O
